Src/Plugins/Security/SEC101_038.PostgreSqlCredentialsValidator.cs

```diff
@@ -70,14 +86,14 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(Dictionary<
 
             var fingerprint = new Fingerprint()
             {
-                Id = id.Value,
+                Id = idValue,
                 Host = hostValue,
                 Port = port?.Value,
                 Secret = secret.Value,
                 Resource = resource?.Value,
             };
 
-            SharedUtilities.PopulateAssetFingerprint(hostValue, ref fingerprint);
+            SharedUtilities.PopulateAssetFingerprint(AzureHosts.ToList(), hostValue, ref fingerprint);
```

Comment:
>AzureHosts [](http://example.com/codeflow?start=53&length=10)

This confuses me, I didn't think this helper accepts a list/enumerable, but only a single value. Is this code compiling?

In any case, we wouldn't want to pass all possible values from AzureHosts, we should be passing the specific host for this finding, which is in `hostValue`. So i think you should revert the change on this line. ?

Reply:
An overload for `populateassettfingerprint` was added in #505 that takes in an additional list of hosts. (additional tests also included in that PR)